cmd/integration-cip-skidspar/models.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+type Status struct {
```

Comment:
This type should ideally be encapsulated in an anti corruption layer, and not published like this. At least put the type declaration in the file that owns it.